.babelrc

```diff
@@ -1,11 +1,20 @@
 {
 	"presets": [
-		"es2015",
+		[ "env", {
+			"targets": {
+				"browsers": [
+					"last 2 versions",
+					"Safari >= 9",
```

Comment:
Wondering if there was a reason we'd special cased Safari previously, since why would we need this line if we're already defining `last 2 versions` ?